lib/src/cpp/common/LocatorList.cpp

```diff
@@ -28,103 +28,118 @@ namespace common {
 namespace locator_list {
 
 std::string print(
+        eprosima::qosprof::utils::ParseXML& manager,
```

Comment:
As the current namespace is within `eprosima::qosprof`, it would be enough with

```suggestion
        utils::ParseXML& manager,
```